src/assets/src/components/queue.tsx

```diff
@@ -340,17 +350,13 @@ export function QueuePage(props: PageProps<QueuePageParams>) {
         await api.removeMeeting(queue!.my_meeting!.id);
     }
     const [doLeaveQueue, leaveQueueLoading, leaveQueueError] = usePromise(leaveQueue);
-    const confirmLeaveQueue = () => {
-        dialogRef.current!.show({
-            title: "Leave Queue?",
-            body: "The queue is closed, but you are still in line. If you leave now, you will not be able to rejoin until the queue is reopened.",
-            actions: [
-                Dialog.CancelAction(),
-                Dialog.OKAction(() => {
-                    doLeaveQueue();
-                }),
-            ],
-        });
+    const confirmLeaveQueue = (queueStatus: 'open' | 'closed') => {
+        const description = 'Are you sure you want to leave the queue? ' + (
+            queueStatus === 'open'
+                ? "You will lose your place in line."
+                : "The queue is currently closed. If you leave now, you will not be able to rejoin until the queue is reopened."
```

Comment:
@mfldavidson or @teshsopher , I'm looking for feedback on the language here, since I tweaked it a bit. I can change it if desired.

When leaving queue with status open: "Are you sure you want to leave the queue? You will lose your place in line."
When leaving queue with status closed: "Are you sure you want to leave the queue? The queue is currently closed. If you leave now, you will not be able to rejoin until the queue is reopened."

Reply:
For the queue with status open, it may be good to say, "By leaving the queue, you will lose your place in line."
For the queue with status closed: "The queue is currently closed. By leaving the queue, you will lose your place in line and will not be able to rejoin until the queue is reopened."

I'm okay with keeping the "Are you sure you want to leave the queue?" in either case. The message mostly should emphasize the action that's being taken and what will happen as a result of the action.

Reply:
Okay, I will change the language to what you specified. I'm curious, do you think the button should be "Leave the queue" or "Leave the line"? Seems like the "Are you sure you want to **blank**?" should be aligned with the button action.

Reply:
I updated the language as requested: https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/192/commits/30dc539f1da0c0353c39a7e87930ecfcf4eea8cb